Lib/test/test_pty.py

```diff
@@ -75,6 +76,14 @@ def _readline(fd):
 
 def expectedFailureIfStdinIsTTY(fun):
     # avoid isatty() for now
+    PLATFORM = platform.system()
+    if PLATFORM == "Linux":
+        os_release = pathlib.Path("/etc/os-release")
+        if os_release.exists():
+            # Actually the file has complex multi-line structure,
+            # these is no need to parse it for Gentoo check
```

Comment:
See https://bugs.python.org/issue28468

Reply:
@vstinner @tiran I happy to use `freedesktop_os_release()` when it lands but want to merge this PR ASAP to fix the broken buildbot.

Reply:
> @vstinner @tiran I happy to use freedesktop_os_release() when it lands but want to merge this PR ASAP to fix the broken buildbot.

Sure.